scripts/entrypoint-dispatcherd.sh

```
@@ -1,31 +1,2 @@
-echo "  Timeout: ${DISPATCHERD_TIMEOUT:-3600}s"
-echo "  Max Tasks: ${DISPATCHERD_MAX_TASKS:-100}"
-echo "  Log Level: ${DISPATCHERD_LOG_LEVEL:-INFO}"
-echo ""
-echo "════════════════════════════════════════════════════════════════"
-echo ""
-
-run_dispatcherd --workers="${DISPATCHERD_WORKERS:-2}" \
-    --timeout="${DISPATCHERD_TIMEOUT:-3600}" \
-    --max-tasks="${DISPATCHERD_MAX_TASKS:-100}" \
-    --log-level="${DISPATCHERD_LOG_LEVEL:-INFO}"
+# Entrypoint for dispatcherd container (dispatcherd via supervisord)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)